quickcomm/forms.py

```diff
@@ -61,3 +61,20 @@ def save(self, author):
 class CreateLoginForm(forms.Form):
     display_name = forms.CharField(max_length=100)
     password = forms.CharField(widget=forms.PasswordInput())
+    
+    
+    
+class EditProfileForm(forms.Form):
```

Comment:
Can you confirm that we don't need the register form anymore?

Reply:
Yes, on main we now use a `UserCreationForm` from `django.contrib.auth.forms` so it is no longer required.